### PR TITLE
Clean up build process and documentation

### PR DESCRIPTION
**Pull request by zaneb**
Eliminate the code dealing with environment variables, and remove documentation about them from the README.

### CODE REVIEW

File: data/ignition/files/usr/local/bin/create-cluster-and-infra-env.sh [hunk 1]
```diff
@@ -9,16 +9,16 @@ source common.sh
...
-CLUSTER_CREATE_OUT=$(curl -X POST -H "Content-Type: application/json" -d '{{.ClusterCreateParamsJSON}}' {{.ServiceBaseURL}}/api/assisted-install/v2/clusters)
+set -e
+
+CLUSTER_CREATE_OUT=$(curl -s -S -X POST -H "Content-Type: application/json" -d '{{.ClusterCreateParamsJSON}}' {{.ServiceBaseURL}}/api/assisted-install/v2/clusters)
 echo "cluster create response: $CLUSTER_CREATE_OUT"
 # pick cluster_id out from cluster create response
-CLUSTER_ID=$(echo $CLUSTER_CREATE_OUT | jq .id)
-# trim quotes from CLUSTER_ID
-CLUSTER_ID=${CLUSTER_ID//\"}
+CLUSTER_ID=$(echo $CLUSTER_CREATE_OUT | jq -r .id)
```
Comment (pawanpinjarkar) [inline data/ignition/files/usr/local/bin/create-cluster-and-infra-env.sh:17]: :+1:

**Review by pawanpinjarkar — APPROVED** [review]
Looks good to me.

**Comment by pawanpinjarkar** [comment]
Suggestion: Please rebase the commits into 1 and update the final commit message.

**Comment by zaneb** [comment]
> rebase the commits into 1

That's not how I roll :slightly_smiling_face:

Each commit is an independent change that could be reverted independently if required, and that's as it should be.

**Pull request merged**